ORM/DataObject.php

```diff
@@ -589,7 +618,7 @@ public function defineMethods() {
 			}
 		}
 
-		if($this->class == 'DataObject') return;
+		if($this->class == 'SilverStripe\ORM\DataObject') return;
```

Comment:
Why, it's just a string comparison. is_a would return true for subclasses, which is not what we want.